src/providers/openai-responses/adapter.ts

```diff
@@ -99,7 +99,7 @@ function fromPromptlContentToOpenAIResponseContent(
           file_data: toOpenAiFileData(item),
         })
         return acc
-      } else if (ContentType.toolCall) {
```

Comment:
el buen bug

Reply:
Add tests for the fix. Luckily we have tests here i think